packages/gatsby/src/commands/develop-process.ts

```diff
@@ -66,6 +84,8 @@ process.on(`message`, msg => {
   }
 })
 
+const bootstrapSpan = tracer.startSpan(`bootstrap`)
```

Comment:
Could we move this inside some function so just importing this module doesn't cause side effect (starting span)?

Reply:
Good point. I'll move it inside the main function